lib/cc/engine/analyzers/java/main.rb

```diff
@@ -0,0 +1,35 @@
+# frozen_string_literal: true
+
+require "flay"
+require "json"
+require "cc/engine/analyzers/reporter"
+require "cc/engine/analyzers/analyzer_base"
+require "cc/engine/processed_source"
+require "cc/engine/sexp_builder"
+
+module CC
+  module Engine
+    module Analyzers
+      module Java
+        class Main < CC::Engine::Analyzers::Base
+          LANGUAGE = "java".freeze
+          PATTERNS = ["**/*.java"].freeze
+          DEFAULT_MASS_THRESHOLD = 40
+          POINTS_PER_OVERAGE = 10_000
```

Comment:
will we tune these eventually or feel happy? curious about how we chose these starting out?

Reply:
I'm sure we'll tune these but I think it'll be easier to flesh out the examples in the QM engines-test repo, and test on .com with forked Java repos then trying to figure out the right values locally during this initial PR.